backend/ipp.c

```diff
@@ -3403,8 +3419,24 @@ run_as_user(char       *argv[],		/* I - Command-line arguments */
   xpc_dictionary_set_int64(request, "command", kPMWaitForJob);
   xpc_dictionary_set_fd(request, "stderr", 2);
 
-  response = xpc_connection_send_message_with_reply_sync(conn, request);
+  sem      = dispatch_semaphore_create(0);
+  response = NULL;
+
+  xpc_connection_send_message_with_reply(conn, request,
```

Comment:
xpc_connection_send_message_with_reply_sync is supported on the same exact Os versions as xpc_connection_send_message_with_reply. Apple recommends the former, hence why I switched to it.